sample-app/screens/HomeScreen.js

```diff
@@ -119,10 +175,66 @@ export default class HomeScreen extends React.Component<Props, State> {
 
   async makeApplePayPayment(isPayment: boolean = true) {
     const title = isPayment ? "Apple Pay payment" : "Apple Pay pre-auth";
+    applePayOptions.transactionType = isPayment ? ApplePayTransactionType.Payment : ApplePayTransactionType.PreAuth;
+    paymentOptions.paymentMethods = PaymentMethodType.AppleGooglePay;
+    try {
+      let response = await RNJudo.showPaymentMethods(
```

Comment:
this is not `makeApplePayPayment`.  please leave it as is to demonstrate how apple pay and google pay works separately without the "payment methods" screen

Reply:
Not sure why you're showing the payment methods screen when this is supposed to make an Apple payment directly. Please see my previous comment.

Reply:
There are two options - make apple payment when we tap a ApplePay button on the home screen and make apple payment when we open payments methods screen.

Reply:
Ok, I just don't see the point of showing a "Select payment methods" screen with one button on it. Is it something anyone would want? Or more importantly, is it something we want to show that possible? Maybe, but probably not.

Reply:
Yes, there was requirements for this. @stpopa could you please confirm that?

Reply:
> Yes, there was requirements for this. @stpopa could you please confirm that?

I don't think there was requirements on this, it's more of a side effect, this is the current behaviour of Android/iOS SDKs, if we want to change it, we need to do it in native SDK's first

Reply:
I understand it's possible to open the "Payment Methods" screen with just one option. My question was more around - is this something we want to demonstrate.
I think both the 'official native' and the normal Apple/Google Pay buttons should just do the same thing - do the appropriate payment straight. I'd leave the "Payment Methods" button to demonstrate that capability and not mix it in with Google/Apple Pay buttons.